### PR TITLE
DASH-1110 Implement nameLookup functionality. Update version to 1.1.0

### PR DESCRIPTION
**Pull request by trukhilio**

### CODE REVIEW

**Review by sardonyxwt — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fioprotocol/fio-wallet-snap",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "The FIO Wallet SNAP lets every Metamask user self-custody their FIO identity and gain access to the FIO App without setting up new accounts or passwords. The FIO App lets MetaMask users register and link their FIO Handle, as well as send and receive FIO Requests and sign NFTs.",
```
Comment (pawmmm) [inline package.json]: Change to "The FIO Wallet SNAP lets every Metamask user self-custody their FIO identity and gain access to the FIO App without setting up new accounts or passwords. The FIO App lets MetaMask users register and link their FIO Handle, as well as send and receive FIO Requests and sign NFTs. FIO Wallet SNAP also supports FIO Handle resolution inside MetaMask."

File: snap.manifest.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "The FIO Wallet SNAP lets every Metamask user self-custody their FIO identity and gain access to the FIO App without setting up new accounts or passwords. The FIO App lets MetaMask users register and link their FIO Handle, as well as send and receive FIO Requests and sign NFTs.",
```
Comment (pawmmm) [inline snap.manifest.json]: Change to "The FIO Wallet SNAP lets every Metamask user self-custody their FIO identity and gain access to the FIO App without setting up new accounts or passwords. The FIO App lets MetaMask users register and link their FIO Handle, as well as send and receive FIO Requests and sign NFTs. FIO Wallet SNAP also supports FIO Handle resolution inside MetaMask."

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fioprotocol/fio-wallet-snap",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (pawmmm) [inline package.json]: Change to 1.1

File: snap.manifest.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
 {
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (pawmmm) [inline snap.manifest.json]: Change to 1.1

File: snap.manifest.json [hunk 5]
```diff
@@ -29,7 +28,25 @@
...
+        "eip155:8453",
+        "eip155:42161",
+        "eip155:42220",
+        "eip155:43114",
+        "eip155:59144",
+        "eip155:7777777"
+      ]
+    }
   },
   "manifestVersion": "0.1"
```
Comment (pawmmm) [inline snap.manifest.json:51]: Should we change to 0.2?

File: src/utils/nameLookup.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    const isDomainValid = validateFioHandle(domain);
+    const { chainCode, tokenCode } = CHAIN_CODES_AND_TOKEN_CODES_BY_NETWORK[chainId] || {};
+
+    if (!chainCode || !tokenCode || !isDomainValid) {
+      return null;
+    }
+
+    try {
+      const publicAddress = await getPublicAddressByFioHandle({
+        apiUrl: 'https://fio.blockpane.com',
```
Comment (pawmmm) [inline src/utils/nameLookup.ts]: Should be https://chain.fio.net
Also should it not be moved to constants.ts?

**Review by pawmmm — APPROVED** [review]

File: snap.manifest.json (hunk 5 above)
Comment (trukhilio) [reply snap.manifest.json:51]: No, we shouldn't.

File: src/utils/nameLookup.ts (hunk 6 above)
Comment (trukhilio) [reply src/utils/nameLookup.ts]: That was test version. Updated to https://chain.fio.net

File: snap.manifest.json (hunk 2 above)
Comment (trukhilio) [reply snap.manifest.json]: Done

File: snap.manifest.json (hunk 4 above)
Comment (trukhilio) [reply snap.manifest.json]: Done. Changed to 1.1.0

File: package.json (hunk 3 above)
Comment (trukhilio) [reply package.json]: Done. Changed to 1.1.0

File: package.json (hunk 1 above)
Comment (trukhilio) [reply package.json]: Done.

**Pull request merged**